POEApi.Model/Socket.cs

```diff
@@ -10,5 +10,27 @@ internal Socket(JSONProxy.Socket s)
             this.Attribute = s.Attr;
             this.Group = s.Group;
         }
+
+        /// <summary>
+        /// Convert the Socket attribute to something consumable by path of building
+        /// </summary>
+        /// <returns></returns>
+        public string ToPobFormat()
+        {
+            switch (Attribute)
+            {
+                case "S":
+                    return "R";
```

Comment:
I'd rather see constants defined with these values instead of having magic strings.